Makefile

```diff
@@ -7,9 +7,16 @@ OPEN_APP ?= xdg-open
 .PHONY: fetch-protos doc help up up-container up-local stop status top kill down proxy-info proxy
 
 # Check http://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
-fetch_protos: ## Fetch protobuf files
+fetch_protos: ##  Fetch protobuf files
 	$(Q) git submodule update --init --recursive
 
+update_protos: ##  Update Protobuf files
```

Comment:
Need to add `.PHONY` for these

Reply:
`fetch_protos` already has its entry above, the one missing is `wasm_build` below.

Reply:
scratch that, I see it's done